.github/workflows/deploy-mainnet.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
           POSTGRES_DB: ${{ secrets.POSTGRES_DB }}
           POSTGRES_DATA: ${{ secrets.POSTGRES_DATA }}
           # hapi
-          HAPI_EOS_API_ENDPOINT: 'https://eos.api.eosnation.io'
+          HAPI_EOS_API_ENDPOINTS: '["https://eos.edenia.cloud","https://eos.api.eosnation.io","https://eos.greymass.com","https://eos.eosphere.io","https://api.eosrio.io"]'
```

Comment:
i find the aloha EOS endpoint to be very good and dosen't rate limit as much. can we add it to the list: https://api.main.alohaeos.com